reference/src/representation/pointers.md

```diff
@@ -0,0 +1,44 @@
+# Representation of reference and pointer types
+
+### Terminology
+
+Reference types are types of the form `&T` or `&mut T`.
+
+Raw pointer types are types of the form `*const T` or `*mut T`.
+
+### Representation
+
+The alignment of reference and raw pointer types is the word size.
+
+The sizes of `&T`, `&mut T`, `*const T` and `*mut T` are the same,
+and are at least one word.
+
+* If `T` is a trait, then the size of `&T` is two words.
```

Comment:
Is this `true` ? I thought there was a mention somewhere that we might want to allow for bigger traits in the future (and this would prevent that).

Reply:
For multi-trait objects like `dyn Foo + Bar` (where neither is a marker trait), yeah. Arguably that would not fall under "`T` is **a** trait". By the same argument, the current wording would exclude e.g. `dyn Any+Send`. Would be good to explicitly note this limitation of the guarantee.

Reply:
Done.